BuildTasks/PublishExtension/v5/task.json

```diff
@@ -0,0 +1,310 @@
+{
+  "id": "631511B4-50AB-47C8-B766-7AE2AA672733",
+  "name": "PublishAzureDevOpsExtension",
+  "friendlyName": "Publish Extension",
+  "description": "Publish an Azure DevOps extension to the Visual Studio Marketplace",
+  "author": "Microsoft Corporation",
+  "helpMarkDown": "",
+  "category": "Deploy",
+  "version": {
+    "Major": 5,
+    "Minor": 0,
+    "Patch": 0
+  },
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [
+    "npm"
+  ],
+  "minimumAgentVersion": "2.206.1",
+  "groups": [
+    {
+      "name": "manifest",
+      "displayName": "Extension manifest",
+      "isExpanded": true
+    },
+    {
+      "name": "advanced",
+      "displayName": "Advanced",
+      "isExpanded": false
+    },
+    {
+      "name": "backcompat",
+      "displayName": "Backward Compatibility",
+      "isExpanded": false
+    }
+  ],
+  "instanceNameFormat": "Publish Extension",
+  "inputs": [
+    {
+      "name": "connectTo",
+      "type": "radio",
+      "label": "Connect to",
+      "required": true,
+      "defaultValue": "AzureRM",
```

Comment:
This is hairy. Since the YAML will not contain the default value people upgrading will end up with broken tasks.

For backwards compat it's probably better to keep the current default.

Reply:
Hmm maybe not... Looks like because this is set to `required:true` current users will have `vsteam` as their default.